kahenya-anita/Bumble-Clone
# Home page anita kahenya

## Description
Changed the folder structure to use the root files for HTML & CSS.

## Ticket [Home Page](https://sharing.clickup.com/36234712/t/h/8669fja70/HS255GMQBGRDP6P)
## Types Of Changes
[] Bug fix

[] New Feature

[x] Breaking Changes

## Checklist:
[x]My code follows the code style of this project

[x]All new and existing tests passed.

[x]My code has been linted and formatted correctly.

[x]I have commented my code, particularly in hard-to-understand areas.

[x]My changes generate no new warnings.

## How can this code be tested?
1. After Cloning this repo , run `git pull
` to ensure you are up to date. 
2. Checkout to [Home-Page_Anita-Kahenya](Home-Page_Anita-Kahenya) 

![Screenshot 2023-03-12 at 8 12 29 PM](https://user-images.githubusercontent.com/62019551/224560785-3cab4e1b-12aa-4e85-ad4b-218c1d348e0d.png)

4. Run the changes in the browser
![Screenshot 2023-03-12 at 8 03 57 PM](https://user-images.githubusercontent.com/62019551/224560325-372bab26-9721-4697-a955-c741a6470844.png)
